pkg/config/constants.go

```diff
@@ -37,6 +37,11 @@ const (
 	SidecarDefaultMemoryLimit = "128M"
 	PVCStorageSize            = "1Gi"
 
+	// WorkspaceStartOnlyLabel is used to specify that the DevWorkspace should not be reconciled further once
+	// it has entered the "Running" phase. This is to enable development of the DevWorkspace operator from within
+	// a DevWorkspace. This label is ignored when `devworkspace.experimental_features_enabled` is set to false.
+	WorkspaceStartOnlyLabel = "controller.devfile.io/development.start_only"
```

Comment:
I'm not sure I understand how we are going to use start_only in development. Start workspace, run local controller, expect that local controller is going to reconcile the workspace where we work?
How then we will test webhooks/workspace start?

I see some annotations or even labels (to easier exclude it from webhooks, but annotations are designed for such things, like we have restricted-access) that allows excluding devworkspace CR from attention all namespaces operator.
Then workflow is like the following:
1. We create a workspace with devworkspace operator project.
2. It's reconciled by all namespace operator.
3. We do changes;
4. We run local operator.
5. We test our changes on the `managed_by_local_operator` DevWorkspace CR.

Reply:
I think I came to workflow you had in mind with start_only:
1. make install
2. kc apply -f ./samples/devworkspace-controller -< with start_only
3. make inject_kubeconfig
4. kc patch deployment/devworkspace-controler-manager --patch "{\"spec\":{\"replicas\":0}}"`
5. Make changes in the workspace.
6. make run now from the workspace.

Such development workspace is like a space shuttle that leaves unneeded engines and fuel tanks )

Reply:
That's pretty much my idea at the moment -- you still have to bootstrap your initial workspace load, sadly.

One flow we could also enable, if we figured out building images within the cluster, would be to not patch the controller to stop it and instead rollout new deployments from within the cluster itself. The annotation is mostly here as a safety net -- we don't want a development bug in our controller killing the workspace being used to develop it.